packages/subgraphs/govern-subgraph/manifest/data/rinkeby-staging.json

```diff
@@ -1,12 +0,0 @@
-{
-  "network": "rinkeby",
-  "dataSources": {
-    "GovernRegistry": [
-      {
-        "name": "GovernRegistry",
-        "address": "0x7a044ff5148220866B9c21E9C6537C7c960eed40",
-        "startBlock": 9932541
-      }
-    ]
-  }
-}
```

Comment:
Why you removed that? btw its look almost exactly what i did in quest subgraph, if i knew here it already was there, i could just copy and paste. You ran in some problems to change that?

Reply:
rinkeby-staging should be the same as rinkeby in my opinion so only one is needed